app/assets/javascripts/discourse/tests/acceptance/user-private-messages-test.js

```diff
@@ -44,27 +44,6 @@ acceptance(
       );
     });
 
-    test("viewing group messages on subfolder setup", async function (assert) {
```

Comment:
Replaced this test with a system spec to avoid stubbing a deprecated private router service